selenium-testing/src/test/java/com/testng/selenium/v1/BaseTest.java

```diff
@@ -0,0 +1,101 @@
+package com.testng.selenium.v1;
```

Comment:
Normally, your package would be relevant to the project you're working on, not the framework used.

E.g. ca.bc.gov.ecc.ofm.selenium.v1 might be a good choice

Reply:
Refactored: https://github.com/bcgov/ecc-ofm/pull/319/commits/180b5b17bf2e8dc5662e9373deea7738980cf48a